nixos/tests/keycloak.nix

```diff
@@ -40,7 +40,7 @@ let
 
           environment.systemPackages = with pkgs; [
             xmlstarlet
-            libtidy
+            html-tidy
```

Comment:
`libtidy` was renamed in https://github.com/midchildan/nixpkgs/commit/cc700ad55b3fc4bc257826b3167c3a1247b50a5b.

Reply:
5bc00cce2223251c6b6b613a3946812a1581cad8 seems like the right one, but it doesn't really matter. We should use `html-tidy`, since that's its real name.